code/modules/economy/economy_machinery/atm.dm

```diff
@@ -0,0 +1,304 @@
+#define ATM_SCREEN_DEFAULT   0
+#define ATM_SCREEN_SECURITY  1
+#define ATM_SCREEN_TRANSFER  2
+#define ATM_SCREEN_LOGS      3
+
+#define PRINT_DELAY  (30 SECONDS)
+#define LOCKOUT_TIME (10 SECONDS)
+
+/obj/machinery/economy/atm
+	name = "Nanotrasen automatic teller machine"
+	desc = "For all your monetary needs! Just insert your ID card to make a withdrawal or deposit!"
+	icon = 'icons/obj/terminals.dmi'
+	icon_state = "atm"
+	anchored = TRUE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 10
+	density = FALSE
+	restricted_bypass = TRUE
+
```

Comment:
Yeah, its still an older feature of econ stuff, I think I may just remove it entirely because its just more overhead transaction tracking that I could care less about until I expand econ features more

Reply:
removed in next commit